src/js/components/TaskView.js

```diff
@@ -168,17 +184,19 @@ class TaskView extends mixin(SaveStateMixin) {
 
     return (
       <div className="flex-container-col flex-grow">
-        <span className="h4 text-align-left flush-top">
+        <span className={headerClassSet}>
           {this.getHeaderText(tasks)}
         </span>
         <FilterBar>
           <FilterInputText
             className="flush-bottom"
             searchString={state.searchString}
             handleFilterChange={this.handleSearchStringChange}
-            inverseStyle={false} />
+            inverseStyle={this.props.inverseStyle} />
```

Comment:
At the moment TaskViews are inverse in nodes/task-view and not inverse in services/task-view, but will soon both be inverse, so this is temporarily accounting for both scenarios